client/login/wp-login/login-links.jsx

```diff
@@ -247,6 +257,40 @@ export class LoginLinks extends Component {
 		);
 	}
 
+	renderQrCodeLoginLink() {
+		if ( ! config.isEnabled( 'login/qr-code-login' ) || this.props.twoFactorAuthType ) {
+			return null;
+		}
+		if ( this.props.isLoggedIn ) {
+			return null;
+		}
+
+		// jetpack cloud cannot have users being sent to WordPress.com
+		if ( isJetpackCloudOAuth2Client( this.props.oauth2Client ) ) {
+			return null;
+		}
+
+		// @todo Implement a muriel version of the email login links for the WooCommerce onboarding flows
```

Comment:
Should this mention the qr code login rather than email?

Reply:
I think removing the link from any oauth2flow, for now, would make sense.